src/augeas_sym.version

```diff
@@ -69,4 +69,6 @@ AUGEAS_0.18.0 {
 AUGEAS_0.20.0 {
     global:
       aug_escape_name;
+      luaopen_augeas;
+      aug_lua;
```

Comment:
You're right, that was a testing thing. I guess this entry also depends on whether we expose aug_lua